code/api/src/modules/subscription/resolvers.js

```diff
@@ -58,4 +58,16 @@ export async function remove(parentValue, { id }, { auth }) {
   } else {
     throw new Error('Access denied.')
   }
-}
\ No newline at end of file
+}
+
```

Comment:
So all mutations require authentication before they are allowed

Reply:
for the mutations that regard a specific user, yeah well want to authorize them before they can make any update ( i believe) i'm not sure how it works with the front end but i think it checks to see if they have a token stored in their cookies

Reply:
cool that makes sense

Reply:
![](https://media0.giphy.com/media/StRigWzyNVcZL74szw/giphy.gif?cid=5a38a5a2nte5nlh8qs1lkow7ncz9jf9nfzf45e2xh57zwqda&rid=giphy.gif)